pyvista/plotting/plotting.py

```diff
@@ -3864,6 +3912,39 @@ def reset_camera_clipping_range(self):
         """Reset camera clipping planes."""
         self.renderer.ResetCameraClippingRange()
 
+    def add_light(self, light, only_active=False):
+        """Add a Light to the scene.
+
+        Parameters
+        ----------
+        light : Light or vtkLight
+            The light to be added.
+
+        only_active : bool
+            If ``True``, only add the light to the active renderer. The default
+            is that every renderer adds the light. To add the light to an arbitrary
+            renderer, see the ``add_light`` method of the Renderer class.
+
+        """
+        # TODO: is this design OK?
```

Comment:
Yes, adding a light to each render makes sense as this is the default behavior for our other methods.  We still need an example for this method.
```suggestion
```